next-env.d.ts

```diff
@@ -1,3 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/types/global" />
 /// <reference types="next/image-types/global" />
+
+// NOTE: This file should not be edited
+// see https://nextjs.org/docs/basic-features/typescript for more information.
```

Comment:
It seems this was auto generated by next, when I started it locally

Reply:
Yep, makes sense. Looks like I forgot to commit this file after upgrading to Next 11.1